modules/swagger-models/src/test/java/io/swagger/models/parameters/SerializableParameterTest.java

```diff
@@ -0,0 +1,62 @@
+package io.swagger.models.parameters;
+
+import java.lang.reflect.Method;
+import java.util.Arrays;
+import java.util.EnumMap;
+import java.util.List;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import io.swagger.models.properties.Property;
+import io.swagger.models.properties.PropertyBuilder;
+import io.swagger.models.properties.PropertyBuilder.PropertyId;
+
+public class SerializableParameterTest {
+	
+	@Test
+	public void testParameters() throws NoSuchMethodException, SecurityException {
+		
+		String[] requredMethods = {
```

Comment:
Let's wrap this array with `@org.testng.annotations.DataProvider` so we could check every property independently.
As for now if we have more then one skipped properties the test will detect the first one only